sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/ModelRepoRecordedTestBase.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Iot.ModelsRepository.Tests
+{
+    public class ModelRepoRecordedTestBase : RecordedTestBase<ModelRepoTestEnvironment>
```

Comment:
There is only one set of tests that need recording and they are in ResolverIntegrationTests